history-of-the-quakers-v4/original/02-book-6-chapter-15.adoc

```diff
@@ -949,39 +957,17 @@ application being made to the Queen and Counsel, it produced the following order
 by which the purport of this act will appear of a
 nature similar to their former persecuting acts.
 
-At the Court of Kensington, the 11th day of October, 1705,
-
-PRESENT,
-
-His Royal Highness Prince George of Denmark,
-
-Lord Archbishop of Canterbury,
-
-Lord Keeper,
-
-Lord Treasurer,
-
-Lord President,
-
-Duke of Somerset,
-
-Duke of Ormond,
+[.embedded-content-document.legal]
+--
 
-Earl of Ranelagh,
-
-Mr. Boyle,
-
-Mr. Secretary Hedges,
-
-Mr. Secretary Harley,
-
-Lord Chief Justice Holt,
-
-Lord Chief Justice Trevor,
+[.signed-section-context-open]
+At the Court of Kensington, the 11th day of October, 1705,
 
-Mr. Vernon,
+[.old-style]
+=== Present
 
-Mr. Earle.
+[.blurb]
+=== His Royal Highness Prince George of Denmark, Lord Archbishop of Canterbury, Lord Keeper, Lord Treasurer, Lord President, Duke of Somerset, Duke of Ormond, Earl of Ranelagh, Mr. Boyle, Mr. Secretary Hedges, Mr. Secretary Harley, Lord Chief Justice Holt, Lord Chief Justice Trevor, Mr. Vernon, Mr. Earle.
```

Comment:
So I wasn't sure what to do with all this, I did what I thought looked okay, but idk if you wanted the names still in a list.

Reply:
yeah, this is super strange.  I formatted a list almost exactly like this somewhere else, so i'm going to modify this one to match that one, but it's so rare i think it's not even worth you bothering trying to remember it, so I'll take care of it. 👍